### ISSUE TITLE
Sidebar navigation does not take focus on mobile

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#1147]
### Does a discussion already exist on the topic you are interested in, or on a similar topic?

- [X] I verified and it doesn't exist

### What

Within the design system section, only if you are in a mobile viewport (device or zoom), you cannot navigate to the mobile index navscroll. The toggle is ok, you can activate it, but you cannot get to either the back button or the internal items via keyboard. 

### Why

It is a bug. 

### Context

It doesn't seem strictly related to the parent component "navscroll" in Bootstrap Italia. There you can focus internal elements, however not the back button.
